src/content/getting-started/app-platform/_index.md

```diff
@@ -1,6 +1,5 @@
 ---
-linkTitle: Deploy applications using App platform
-title: Using App platform to deploy applications
+title: Deploy applications using the App platform
```

Comment:
but if we remove `linkTitle` front matter fails. Also in some occasion we want to have different menu title than page title, don't you think?

Maybe we can do that when there is no linktitle use title but it will need a change in the script

Reply:
What exactly fails? Hugo source [code](https://github.com/gohugoio/hugo/blob/73779707a2d5fefd4915f8ee2021f265d841dea2/hugolib/page__meta.go#L182) and docs say that it defaults to `.Title` without requiring an explicit `{{ or .LinkTitle .Title }}` in any template.

I don't see anything wrong in the rendered output at the moment. Except for this which seemed to be there before my changes:

<img width="225" alt="Screenshot 2023-06-07 at 14 22 39" src="https://github.com/giantswarm/docs/assets/1376043/c1798ade-46f7-4ca0-9976-3c240c968508">

Reply:
yeah the overview section should not exists, not sure what happened. I will do a PR